composer.json

```diff
@@ -22,7 +22,6 @@
     "php": ">=5.3"
   },
   "require-dev": {
-    "10up/wpacceptance": "dev-master",
```

Comment:
`composer.lock` still contain lot of references to wpacceptance and deps. Consider running `composer update` locally and commit the updated lock file here.

After `composer update && composer install` all tests will fail because PHPUnit 9.5 will be installed by default. Consider locking it with 7.5 in composer.json OR upgrading all tests to be compatible with PHPUnit 9.5.

Reply:
thanks for the suggestion, implemented!